finetrainers/cogvideox/cogvideox_lora.py

```diff
@@ -325,3 +325,18 @@ def _pad_frames(latents: torch.Tensor, patch_size_t: int):
     "forward_pass": forward_pass,
     "validation": validation,
 }
+
+COGVIDEOX_T2V_SFT_CONFIG = {
```

Comment:
Need to have the SFT configs in the `init`s of the model modules. Such as:
https://github.com/a-r-r-o-w/finetrainers/blob/dbffc80e7c7cbe2640ae1e3b53c5a1b50ab8e2d9/finetrainers/cogvideox/__init__.py#L1